### PR TITLE
Warning when stripping master key.

### PR DESCRIPTION
**Pull request by tdjogi010**

### CODE REVIEW

**Comment by tdjogi010** [comment]
Shows a dialogue asking for user's confirmation.Let me know if the message needs to be changed #1736

**Comment by tdjogi010** [comment]
@dschuermann @Valodim it would be great if you can review it so that I can make changes if something is wrong.

**Comment by Valodim** [comment]
Will look at this later today, (sorry for the delay!)

File: OpenKeychain/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -331,6 +333,7 @@
     <string name="public_key_deletetion_confirmation">"Delete key '%s'?"</string>
     <string name="also_export_secret_keys">"Also export secret keys"</string>
     <string name="reinstall_openkeychain">"You encountered a known bug with Android. Please reinstall OpenKeychain if you want to link your contacts with keys."</string>
+    <string name="alert_strip">"Stripping this Subkey will render it useless."</string>
```
Comment (Valodim) [inline OpenKeychain/src/main/res/values/strings.xml]: don't capitalize subkey

File: OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/dialog/EditSubkeyDialogFragment.java [hunk 2]
```diff
@@ -75,7 +75,22 @@ public void onClick(DialogInterface dialog, int which) {
                         sendMessageToHandler(MESSAGE_REVOKE, null);
                         break;
                     case 2:
```
Comment (Valodim) [inline OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/dialog/EditSubkeyDialogFragment.java]: While you're touching this code, can you introduce constants for this switch statement? i.e. `LIST_ITEM_CHANGE_EXPIRY`

File: OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/dialog/EditSubkeyDialogFragment.java [hunk 3]
```diff
@@ -75,7 +75,22 @@ public void onClick(DialogInterface dialog, int which) {
...
-                        sendMessageToHandler(MESSAGE_STRIP, null);
+                        CustomAlertDialogBuilder stripAlertDialog = new CustomAlertDialogBuilder(getActivity());
```
Comment (Valodim) [inline OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/dialog/EditSubkeyDialogFragment.java]: This is a lot of code in a case, please extract it into a method

**Comment by Valodim** [comment]
(I forgot this one again in all the buzz of notifications. Sorry about that.)

**Comment by Valodim** [comment]
Ping. You still working on this?

**Comment by tdjogi010** [comment]
Sorry I am not.

**Comment by Valodim** [comment]
no worries. we'll pick up on this, should be simple to finish. thanks for your work!

**Pull request merged**